description.md

```diff
@@ -17,6 +17,121 @@ Each section details how a specific type of content is to be developed, stored a
+
+- [Moodle](https://moodle.org/) is a free open-source learning platform dedicated to course management.
+It supports graded assignments, quizzes, storing teaching materials and many other educational activities.
+
+- [SmileyTutor (formerly called TutorWeb)]((https://tutor-web.net/)) is another free learning platform whose highlight is that it was built around the [SmyleyCoin cryptocurrency](#cryptocurrency-rewards) as a reward for completing drills and quizzes.
+
+- [tech.io](https://tech.io/) is another quiz platform.
+More simplistic than Moodle or SmileyTutor, it allows teachers to easily create _playgrounds_, which are learning environments designed around specific subjects, made up of reading material, followed by practical exercises and quizzes.
+
+Each of these platforms supports a different syntax for uploading questions.
+To make our quizzes comply with educational platforms easily, we need to store them in a generic format that is easily representable and automatically convertible to other required formats.
+For this reason and for the sake of consistency, we are storing questions in the same Markdown format as all of our written material.
+
+We are developing a modular and easily extendible converter that ports quizzes from Markdown to other formats.
+The format that we use to store quizzes is easily convertible to other formats.
+Therefore, we do not implement converters for all the existing formats.
+Instead, we do provide a program that is easily extendible to support any kind of format.
+
+#### Storage Format
+
+We are using Markdown as a generic storage format for quizzes because it is a text-based format, so it is suitable for written questions.
+Unlike more complex formats like LaTeX, Markdown offers a much simpler syntax which makes quizzes easier to contribute to by educators and students alike.
+
+The format we propose relies on headers to represent categories such as difficulty, topics, question text, feedback.
+Each question has a title written on the first line as a a level 1 header (`#`).
+Each category is specified as a level 2 header (`##`).
+The contents of the category consist of the text beneath the header.
+For multiple-choice questions, correct answers are marked as list entries starting with `+`, while incorrect ones are marked with `-`.
+
+See the example below:
+
+```text
+# Sample Question Name
+
+##  Difficulty
+
+easy
+
+## Tags
+
+- topic-1
+
+- topic-2
+
+## Question Text
+
+Is this a sample question?
+
+## Question Answers
+
++ Yes
+
+- No
+
+- Maybe
+
+## Feedback
+
+The question is self-reflecting and its title is "Sample Question Name", so it is a sample question.
+```
+
+Note that correct answers are marked as list entries starting with `+`, while incorrect ones are marked with `-`.
+In addition, the headings starting with `#` are just keys for different sections of the quiz.
+Therefore, none of them are mandatory given our format and the developer is free to add or remove any of them.
+
```

Comment:
Detail the structure of the `quizzes/` directory as part of each topic.

Reply:
What do you mean by "each topic"?